pytradfri/color.py

```diff
@@ -46,16 +46,16 @@ def supported_features(data: LightResponse) -> int:
     """Return supported features."""
     supported_color_features = 0
 
-    if data.dimmer:
+    if data.dimmer is not None:
```

Comment:
The integer 0 is evaluated as falsy, so we need to check for None explicitly for the attributes that can return integers.